jetty-start/pom.xml

```diff
@@ -44,11 +43,15 @@
             <configuration>
               <createDependencyReducedPom>false</createDependencyReducedPom>
               <createSourcesJar>true</createSourcesJar>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>shaded</shadedClassifierName>
               <filters>
                 <filter>
                   <artifact>org.eclipse.jetty:jetty-util</artifact>
                   <includes>
                     <include>org/eclipse/jetty/util/JavaVersion*</include>
+                    <include>org/eclipse/jetty/util/TopologicalSort*</include>
```

Comment:
To quickly identify accidental dependencies these two above should not use globs, but fully qualified names.